packages/ui5-proxy-middleware/src/base/utils.ts

```diff
@@ -150,17 +145,13 @@ export const getYamlFile = (args: string[]): string => {
  * @returns Path to the webapp folder
  */
 export const getWebAppFolderFromYaml = async (ui5YamlPath: string): Promise<string> => {
-    let webAppFolder = 'webapp';
-
     if (existsSync(ui5YamlPath)) {
-        const ui5Yaml = parseDocument(await promises.readFile(ui5YamlPath, { encoding: 'utf8' })).toJSON();
-
-        if (ui5Yaml.resources?.configuration?.paths?.webapp) {
-            webAppFolder = ui5Yaml.resources.configuration.paths.webapp;
-        }
+        const test = readFileSync(ui5YamlPath, { encoding: 'utf8' });
```

Comment:
Can we have another name for this variable, e.g. `yamlFile`?

Reply:
https://github.com/SAP/open-ux-tools/pull/424/commits/6320c5d2d4d9064ba145e19c12fbbeaf76e7290f